### PR TITLE
Replace `vergen-version` with `daemon::VERSION` constant

### LINKED ISSUE #2697: Incorrect version logged in testnet setup
The following log message

https://github.com/itchysats/itchysats/blob/8dee8d3e93936dbd5f08beb0b782cc6caa971ad4/maker/src/main.rs#L43

works as intended locally e.g.

```
2022-08-16 01:51:13  INFO maker: Running version: 0.5.4-69-g9030093
```

but doesn't work in our testnet setup, as can be seen [on Grafana](https://itchysats.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22grafanacloud-itchysats-logs%22,%22queries%22:%5B%7B%22refId%22:%22B%22,%22editorMode%22:%22builder%22,%22expr%22:%22%7Bcontainer%3D%5C%22maker%5C%22,%20setup%3D%5C%22testnet%5C%22%7D%20%7C%3D%20%60Running%20version%60%20%7C%20json%22,%22queryType%22:%22range%22,%22hide%22:false%7D%5D,%22range%22:%7B%22from%22:%221660586200937%22,%22to%22:%221660592992104%22%7D%7D).

The same behaviour is [true for the taker](https://itchysats.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22grafanacloud-itchysats-logs%22,%22queries%22:%5B%7B%22refId%22:%22B%22,%22editorMode%22:%22builder%22,%22expr%22:%22%7Bcontainer%3D%5C%22taker_master%5C%22,%20setup%3D%5C%22testnet%5C%22%7D%20%7C%3D%20%60Running%20version%60%20%7C%20json%22,%22queryType%22:%22range%22,%22hide%22:false%7D%5D,%22range%22:%7B%22from%22:%221660586200937%22,%22to%22:%221660592992104%22%7D%7D).

These log messages are very important because they help us isolate bugs e.g. we only started seeing this symptom after commit XYZ.

### PR DESCRIPTION
**Pull request by luckysori**
Fixes https://github.com/itchysats/itchysats/issues/2697.

We no longer use git to tell us what version (or patch) we're on. Whilst it was a nifty feature, it didn't always work and it made it
harder to build either of the binaries in foreign environments.

### CODE REVIEW

File: maker/src/actor_system.rs [hunk 1]
```diff
@@ -247,7 +247,7 @@ where
             let identity = identity.libp2p.clone();
             move || {
                 identify::listener::Actor::new(
-                    vergen_version::git_semver().to_string(),
+                    daemon::VERSION.to_string(),
```
Comment (klochowicz) [inline maker/src/actor_system.rs]: maybe we could add a pub method in daemon/lib.rs that just returns `version() -> String` ? this way it's more flexible and less noisy in all the code here.

**Review by klochowicz — COMMENTED** [review]
concept ACK

 is there a plan to auto-bump daemon to prevent it being out-of-sync?
I guess an an intermittent step we could add a manual step in the release process...

**Review by bonomat — APPROVED** [review]
👍

**Comment by bonomat** [comment]
@luckysori : don't you want to merge this? :)

**Comment by luckysori** [comment]
bors retry

**Pull request merged**